### PR TITLE
doc: document all available plug-ins

### PR DESCRIPTION
**Pull request by trusz**
We would like to gather all available plugins from the whole community.

Rendered version: [plugins.md](https://github.com/trusz/open-scd/blob/doc/plugins/docs/plugins.md)

Issue:
- https://github.com/openscd/open-scd/issues/1435

### CODE REVIEW

**Review by pascalwilbrink — COMMENTED** [review]
LGTM!
Small remark from the grammar police:(
Can you change `plugins` into `plug-ins` to make it more consistent across the ecosystem?

File: README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ In order to install your own instance of OpenSCD on your own webserver (e.g. on
 
 If you don't have your own webserver but still want your own version of OpenSCD hosted locally (e.g. on a system without an internet connection), you can [use a browser plugin that acts as a local webserver](https://github.com/openscd/open-scd/wiki/Install-OpenSCD#offline) (only for you) instead.
 
+## Plugins
+
+We gather the available plugins from the community in the [plugins](docs/plugins.md) file.
```
Comment (pascalwilbrink) [inline README.md:24]:
```suggestion
We gather the available plugins from the community in the [plug-ins](docs/plugins.md) file.  
```

File: README.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ In order to install your own instance of OpenSCD on your own webserver (e.g. on
 
 If you don't have your own webserver but still want your own version of OpenSCD hosted locally (e.g. on a system without an internet connection), you can [use a browser plugin that acts as a local webserver](https://github.com/openscd/open-scd/wiki/Install-OpenSCD#offline) (only for you) instead.
 
+## Plugins
```
Comment (pascalwilbrink) [inline README.md:22]:
```suggestion
## Plug-ins
```

File: README.md [hunk 3]
```diff
@@ -19,6 +19,10 @@ In order to install your own instance of OpenSCD on your own webserver (e.g. on
...
+
+We gather the available plugins from the community in the [plugins](docs/plugins.md) file.  
+If you would like to list your plugin here, please open a pull request.
```
Comment (pascalwilbrink) [inline README.md:25]:
```suggestion
If you would like to list your plug-in here, please open a pull request.
```

File: docs/plugins.md [hunk 4]
```diff
@@ -0,0 +1,294 @@
+# Plugins
```
Comment (pascalwilbrink) [inline docs/plugins.md:1]:
```suggestion
# Plug-ins
```

File: docs/plugins.md [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+
+In this file we gather all the plugins that are available in the community.
```
Comment (pascalwilbrink) [inline docs/plugins.md:3]:
```suggestion
In this file we gather all the plug-ins that are available in the community.
```

File: docs/plugins.md [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+We mark our official plugins that are supported by the OpenSCD organization with a badge:
```
Comment (pascalwilbrink) [inline docs/plugins.md:4]:
```suggestion
We mark our official plug-ins that are supported by the OpenSCD organization with a badge:
```

File: docs/plugins.md [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+<kbd>✅ Official</kbd>
+
+> [!CAUTION]
+> We do not guarantee the quality or the security of the non-official plugins
```
Comment (pascalwilbrink) [inline docs/plugins.md:8]:
```suggestion
> We do not guarantee the quality or the security of the non-official plug-ins
```

File: docs/plugins.md [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+
+> [!NOTE]
+> If you want to add a plugin to this list, please open a pull request.
```
Comment (pascalwilbrink) [inline docs/plugins.md:11]:
```suggestion
> If you want to add a plug-in to this list, please open a pull request.
```

File: docs/plugins.md [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+
+> [!TIP]
+> How to add a plugin: [Extensions.md](../packages/open-scd/public/md/Extensions.md)
```
Comment (pascalwilbrink) [inline docs/plugins.md:14]:
```suggestion
> How to add a plug-in: [Extensions.md](../packages/open-scd/public/md/Extensions.md)
```

File: docs/plugins.md [hunk 10]
```diff
@@ -0,0 +1,294 @@
...
+
+## Core Plugins
```
Comment (pascalwilbrink) [inline docs/plugins.md:16]:
```suggestion
## Core Plug-ins
```

File: docs/plugins.md [hunk 11]
```diff
@@ -0,0 +1,294 @@
...
+
+Core plugins are general IEC 61850 Plugins. They can be used in any project.
```
Comment (pascalwilbrink) [inline docs/plugins.md:18]:
```suggestion
Core plug-ins are general IEC 61850 Plug-ins. They can be used in any project. 
```

**Review by pascalwilbrink — COMMENTED** [review]
Sorry for the massive request changes

**Review by pascalwilbrink — APPROVED** [review]

File: docs/plugins.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (trusz) [reply docs/plugins.md]: That is out of the scope of this PR and I would do it in the one where we change the naming everywhere

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (trusz) [reply README.md]: I'd say that the option to remove plugin is implicitly in this description.
Also, I think we can ask it in the PR if necessary.

**Review by pascalwilbrink — APPROVED** [review]
LGTM!

**Pull request merged**